package.json

```diff
@@ -64,11 +60,11 @@
     "docgen": "node --harmony scripts/build_docs.js",
     "clean": "rm -rf dist/npm",
     "typecheck": "flow check",
-    "compile": "babel -D --optional runtime -d dist/npm/ src/",
-    "watch": "babel -w -D --optional runtime -d dist/npm/ src/",
+    "compile": "mkdir -p dist/npm/common && cp -r src/common/schemas dist/npm/common/ && tsc",
```

Comment:
typescript only handles typescript files, so we have to copy the `schemas` directory here ourselves. Normally you could just move this folder to a single place outside of the `src` directory and load them by relative path, but unfortunately that breaks because `dist/npm` & `src/` are at different levels of nesting (`../` vs `../../`).

This doesn't feel *too* complex but if we ever need to add more here we could move this into a build script or gulp file.